src/preprocess/visitor.ts

```diff
@@ -637,6 +637,9 @@ export class Visitor {
 
   // Special handling for CloudFormation `Resource` entries and iidy custom Resource templates.
   visitResourceNode(node: any, path: string, env: Env): AnyButUndefined {
+    if (node instanceof yaml.Tag) {
+      return this.visitResourceNode(this.visitNode(node, appendPath(path, '$Preprocess'), env), path, env);
```

Comment:
Could this use the tag name rather than `$Preprocess` here?

If this is a CloudFormation tag (instead of an iidy one) would `visitNode` essentially be a no-op?

Reply:
1) I suppose so. I'll do that.
2) Yep

Reply:
So if it were a CloudFormation tag, would it produce a loop here?

Reply:
Yeah, it would be a bug in that case anyway. We could check explicitly for `iidy` custom tags and fail hard if it's not one.

Reply:
I just pushed some protection against that.